docs/admin-guide.md

```diff
@@ -17,29 +17,51 @@ To get started, a plugin that is installed on the Mattermost Server is needed to
 
 Now, you'll need to configure your Confluence server to communicate with the plugin on the Mattermost Server. The instructions are different for Cloud vs Server/Data Center. 
 
-#### Set up Confluence Server
+#### Set up Confluence Server for server version less than 9
 
 To get started, type in `/confluence install server` in a Mattermost chat window.
 
-1. Download the [Mattermost for Confluence OBR file](https://github.com/mattermost/mattermost-for-confluence/releases) from the "Releases" tab in the Mattermost-Confluence plugin repository. Each release has some JVM files and an .OBR file.  Download it to your local computer. You will upload it to your Confluence server later in this process. 
-2. Log in as an Administrator on Confluence to configure the plugin.
-3. Create a new app in your Confluence Server by going to **Settings > Apps > Manage Apps**. For older versions of Confluence, go to **Administration > Applications > Add-ons > Manage add-ons**.
-4. Choose **Settings** at the bottom of the page, enable development mode, and apply the changes. Development mode allows you to install apps from outside of the Atlassian Marketplace.
-5. Press **Upload app**.
+1. Provide the Confluence base URL in the first step of the installation
+2. Select **No** in the **Confluence server version greater than or equal to 9** question in second step of the flow manager.
+3. Download the [Mattermost for Confluence OBR file](https://github.com/mattermost/mattermost-for-confluence/releases) from the "Releases" tab in the Mattermost-Confluence plugin repository. Each release has some JVM files and an .OBR file.  Download it to your local computer. You will upload it to your Confluence server later in this process. 
```

Comment:
We have moved the Confluence plugin to the Mattermost org.  I was unaware of this repo linked here.  https://github.com/mattermost-community/mattermost-for-confluence

@fmartingr can we get this one moved as well?